attributes/default.rb

```diff
@@ -96,7 +98,7 @@
   default['php']['fpm_conf_dir']     = '/etc/php5/fpm'
 
   if (platform?('debian') && node['platform_version'].to_i >= 9) ||
-     (platform?('ubuntu') && node['platform_version'].to_f == 16.04)
+     (platform?('ubuntu') && Chef::VersionConstraint.new('= 16.04').include?(node['platform_version']))
```

Comment:
Is there a particular reason for moving these all to the VersionConstraint class? I can't see a reason we can't just use .to_i and .to_f in this scenario.

Reply:
Eh, it seemed like a good idea, but yeah, it’s probably overkill, esp. since they can’t all be checked that way.

I’ll switch ’em back.